maps/PokemonFanClub.asm

```diff
@@ -77,10 +77,9 @@ PokemonFanClubClefairyGuyScript:
 	writetext PokemonFanClubClefairyGuyTakeThisDollBackToGirlText
 	promptbutton
 	waitsfx
-	giveitem LOST_ITEM
+	verbosegiveitem LOST_ITEM
 	iffalse .NoRoom
 	disappear POKEMONFANCLUB_FAIRY
-	writetext PokemonFanClubPlayerReceivedDollText
```

Comment:
For now I suggest keeping these in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.